lib/elastic_apm/instrumenter.rb

```diff
@@ -195,7 +195,7 @@ def set_tag(key, value)
       return unless current_transaction
 
       key = key.to_s.gsub(/[\."\*]/, '_').to_sym
```

Comment:
We could save a string allocation by using `gsub!` instead. Could be too insignificant to worry about. How does `gsub!` even work with `frozen_string_literal`? Questions, questions.

Reply:
You can't call `gsub!` on a frozen string, unfortunately. I can't think of another way to do this without creating another String. Let me know if you have any ideas...

Reply:
We could do it if we wrote this in a c extension ; )